spec/factories/loans.rb

```diff
@@ -55,11 +55,12 @@
 
 
     trait :active do
-      status_option_id Loan::STATUS_OPTIONS.value_for('Active')
+      # note: can't seem to count on the seed.db data having been loaded when these are executed
+      status_option_id 1 #Loan.status_option_set.value_for_label('Active')
```

Comment:
Yeah.  I hear you.  The functional implementation was easy enough.  But the unit testing has been a can of worms.  I'll let you and @linesarefuzzy sync up and decide what makes the most sense.